src/core/installAddon.js

```diff
@@ -223,11 +226,12 @@ const userAgentOSToPlatform = {
  *
  * type FindInstallUrlParams = {|
  *   installURLs: $PropertyType<AddonType, 'installURLs'>,
+ *   src?: string,
  *   userAgentInfo: UserAgentInfoType,
  * |};
```

Comment:
Why is this in a comment btw?

Reply:
Because converting this file to Flow would be an arduous task